svfs/swift.go

```diff
@@ -82,7 +82,7 @@ func getMtime(object *swift.Object, headers swift.Headers) time.Time {
 		}
 	}
 	if object.LastModified.IsZero() {
-		return time.Now()
+		return MountTime
```

Comment:
If you modifiy the time here, this will be applied not only to directories but all files.